OTKit/README.md

```diff
@@ -19,15 +19,41 @@
 $ npm install --save-dev <token-name>
 ```
 
+### Install postcss-pxtorem for Pixel -> Rem conversion
+All contributions to this project should be in pixels.  For users of these tokens, you will have to use [postcss-pxtorem](https://github.com/cuth/postcss-pxtorem) to handle the pixel to rem conversion in your CSS files.
+
+```
+npm install --save-dev postcss-pxtorem
+```
+
+Then add it to your [postcss](https://github.com/postcss/postcss) processors with the following recommend configuration:
+
+```
+import pxtorem from 'postcss-pxtorem';
+
+const processors = [
+  pxtorem({
+    rootValue: 16,
+    selectorBlackList: [/^html$/],
+    replace: true
```

Comment:
This just omits pixel fallbacks for browsers that don't support rems (pre IE11).  Based on my understanding of what browsers we support at OpenTable, this should be a sane suggestion.